portals/devportal/source/src/app/components/Shared/AppsAndKeys/TokenExchangeKeyConfiguration.jsx

```diff
@@ -591,7 +591,7 @@ const TokenExchangeKeyConfiguration = (props) => {
                                                         </React.Fragment>
                                                     )}
                                                     {showToken &&
-                                                    <ViewToken token={{...tokenResponse, isOauth: true}}/>}
+                                                    <ViewToken token={{...tokenResponse, isOauth: true}} isTokenExchange={true} />}
```

Comment:
Just setting the `isTokenExchange` prop is eneugh
```suggestion
                                                    <ViewToken token={{...tokenResponse, isOauth: true}} isTokenExchange />}
```